src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitForeachTests.cs

```diff
@@ -4872,20 +5439,86 @@ public sealed class Enumerator
 }
 public static class Extensions
 {
-    public static C.Enumerator GetAsyncEnumerator(this C self) => new C.Enumerator();
+    public static C.Enumerator GetAsyncEnumerator(this FormattableString self) => throw null;
+    public static C.Enumerator GetAsyncEnumerator(this object self) => new C.Enumerator();
 }";
             var comp = CreateCompilationWithMscorlib46(source, options: TestOptions.DebugExe, parseOptions: TestOptions.RegularPreview);
             comp.VerifyDiagnostics();
             CompileAndVerify(comp, expectedOutput: "123");
         }
 
         [Fact]
-        public void TestGetAsyncEnumeratorPatternViaExtensionsWithUpcast()
+        public void TestGetAsyncEnumeratorPatternViaExtensionsWithFormattableStringConversion2()
```

Comment:
>TestGetAsyncEnumeratorPatternViaExtensionsWithFormattableStringConversion2 [](start = 20, length = 74)

Can you please add a test where we actually use the conversion to `FormattableString`? Or is that not possible for some reason?